pages/learn/manage/actions.md

```diff
@@ -88,6 +88,10 @@ This action allows you to remotely update the host OS running on your device. Fo
 
 Turning on local mode is useful when you are prototyping your application, as it allows you to push changes to your device over the local network without relying on the {{ $names.company.lower }} build pipeline. You can find more information in our [development guide][local-mode].
 
+### Deactivate Device
+
+This action will [deactivate the device][inactive-devices] and charge a one-time deactivation fee. Note that the device must be offline to be able to deactivate the device.
```

Comment:
outside the scope here but i'm kind of curious the heuristic you use for inline notes vs special blocked notes

Reply:
That's a good observation and hadn't given it much thought and using in the colloquial sense. I'll actually remove the "Note that" as "The device must be offline to be able to deactivate the device." is correct.